index.html

```diff
@@ -150,13 +141,14 @@ <h2>EDX Free Online Courses</h2>
 				<article>
 					<h2>Vrije Universiteit Amsterdam, Amsterdam, The Netherlands</h2>
                     <p class="subDetails">August 2011 - May 2013</p>
-					<p>Completed 87 credits towards a Bachelor of Science in Mathematics.</p>
+					<p>Completed 87 credits towards a Bachelor of Science in Mathematics.<br />

```

Comment:
People in US don't know what this means in the Dutch system. What's the equivalent in a math degree here? It's ~2/3rds, yes? So 5 semesters? Equivalent of a Math Minor?